src/core/acroform/PDFAcroField.ts

```diff
@@ -96,6 +110,24 @@ class PDFAcroField {
     return DA?.asString();
   }
 
+  setFontSize(fontSize: number) {
```

Comment:
Does this function work for newly created text fields? E.g. https://pdf-lib.js.org/docs/api/classes/pdfform#createtextfield. It's important that form methods work for all types of fields.

Reply:
Yes, I've also implemented integration tests for this. The node one is here: https://github.com/duffyd/pdf-lib/blob/efbc0ee4da21d7cb6a4ed4f61859326148449e8b/apps/node/tests/test17.ts#L110-L113